front/src/service/AuthenticationService.js

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios';
+
+const API_URL = 'http://localhost:8080';
+export const USER_NAME_SESSION_ATTRIBUTE_NAME = 'authenticatedUser';
+
+class AuthenticationService {
+
+    executeAuthenticationService(username, password) {
+        return axios.get(`${API_URL}/auth`,
+            {headers: {authorization: this.createBasicAuthToken(username, password)}});
+    }
+
+    createBasicAuthToken(username, password) {
+        return 'Basic ' + window.btoa(username + ":" + password);
+    }
+
+    registerSuccessfulLogin(username, password) {
+        sessionStorage.setItem(USER_NAME_SESSION_ATTRIBUTE_NAME, username);
+        this.setupAxiosInterceptors(this.createBasicAuthToken(username, password));
+        document.getElementById("sign-in").hidden = true;
+        let loginLink = document.getElementById("login");
+        loginLink.innerHTML = "Log Out";
+        loginLink.addEventListener("click", this.logout)
+    }
+
+    isUserLoggedIn() {
+        let user = sessionStorage.getItem(USER_NAME_SESSION_ATTRIBUTE_NAME);
```

Comment:
`sessionStorage` may be changed manually without loging. Everyone can see Home Page without loging in

Reply:
yes, it's basic approach. Anyway without logging in there is no access to backend so you can see home page but without any content